src/backend/tests/routers/test_chat.py

```diff
@@ -101,6 +101,10 @@ def test_fail_chat_missing_user_id(
         headers={"Deployment-Name": ModelDeploymentName.CoherePlatform},
     )
 
+    import pdb
```

Comment:
Looks like we should remove this

Reply:
definitely, ty

Reply:
For testing, I'll disable auth for now ,I'll have to revisit it later